tests/contrib/scrapy/test_pipelines.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-from unittest import TestCase
+from unittest import mock, TestCase
```

Comment:
`unittest.mock` is not available in Python 2.7
You need to install `mock` library for this Python version and import accordingly.